src/components/PersonLink.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person,
+  selectedPerson: string,
+};
+
+export const PersonLink: React.FC<Props> = ({ person, selectedPerson }) => {
+  const { search } = useLocation();
+
+  const {
+    name,
+    sex,
+    born,
+    died,
+    slug,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames(
+        { 'has-background-warning': slug === selectedPerson },
+      )}
+    >
+      <td>
+        <Link
+          to={`/people/${slug}${search}`}
+          className={classNames({ 'has-text-danger': sex === 'f' })}
+        >
+          {name}
+        </Link>
+      </td>
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {!motherName && '-'}
+
+        {motherName && !mother && motherName}
+
+        {motherName && mother && (
+          <Link
+            className="has-text-danger"
+            to={`/people/${mother.slug}${search}`}
+          >
+            {motherName}
+          </Link>
+        )}
+      </td>
+      <td>
+        {!fatherName && '-'}
+
+        {fatherName && !father && fatherName}
+
+        {fatherName && father && (
+          <Link to={`/people/${father.slug}${search}`}>
+            {fatherName}
+          </Link>
+        )}
```

Comment:
```suggestion
        {father ? (
          <PersonLink perosn={father} search={search} >
        ) : fatherName || '-'}
```

it can be shorter and more readable. same for mother

Also suggest to make separate component for example PersonLink and use for mother and father